e2e/e2e-values.yml

```diff
@@ -3,9 +3,9 @@ forwarderImage:
   tag: 1.20.5
   repository: newrelic/k8s-events-forwarder
 agentImage:
-  pullPolicy: Never
-  tag: e2e
-  repository: e2e/newrelic-infrastructure
```

Comment:
I think these were meant to be unpullable since we "inject" this image as a previous step for the e2e test, but I'm not sure if this is still the case. Perhaps @paologallinaharbur knows more about if this is still used.

Reply:
Those were unpullable since we were building them and otherwise minikube tries to find them -failing-.
However, if the image is not build we can simply get it from the repo and use it